widget/textinput.go

```diff
@@ -270,6 +273,12 @@ func (o TextInputOptions) TabOrder(to int) TextInputOpt {
 	}
 }
 
+func (o TextInputOptions) SubmitOnEnter(to bool) TextInputOpt {
```

Comment:
Can you please rename "to" to something a bit more descriptive and add a comment for the method?

Reply:
all functions to set properties have the parameter "to"

Reply:
sorry, not all, I can change it to newValue which would make more sense, and none of these functions have comments as well, I can add it, but I just preserve the format of the rest of the code.

Reply:
Hey, no judgement, but while people are there, I am trying to get a little cleaner code =). I think the only one with "to" would be TabOrder but even that "to" is a bad name.